src/HonzaBotner.Discord.Services/Commands/ReminderCommands.cs

```diff
@@ -69,7 +74,7 @@ public ReminderCommands(IRemindersService service, IOptions<ReminderOptions> opt
                 message.ChannelId,
                 datetime.Value, // This is safe, as the nullability is validated above
                 title,
-                content
+                null
```

Comment:
This makes the content field useless and IMO can be then safely removed from the database schema etc.
Any reasons for removing it?

Reply:
+ if I'm correct, the command parameter can be therefore dropped